From 8c9d84f540d1f878958d02bef1d5c221fc8d30c0 Mon Sep 17 00:00:00 2001
From: Toomas Soome <tsoome@me.com>
Date: Wed, 15 Nov 2023 16:13:17 +0200
Subject: [PATCH] 16061 man: qsort has no argument Ibase Reviewed by: Peter
 Tribble <peter.tribble@gmail.com> Reviewed by: Gordon Ross
 <gordon.w.ross@gmail.com> Reviewed by: Andy Fiddaman <illumos@fiddaman.net>
 Approved by: Robert Mustacchi <rm@fingolfin.org>

---
 usr/src/man/man3c/qsort.3c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/usr/src/man/man3c/qsort.3c b/usr/src/man/man3c/qsort.3c
index 63ba24ba91..c6e2e39354 100644
--- a/usr/src/man/man3c/qsort.3c
+++ b/usr/src/man/man3c/qsort.3c
@@ -19,7 +19,7 @@
 .\" Copyright (c) 2002, Sun Microsystems, Inc.  All Rights Reserved
 .\" Copyright 2020 Oxide Computer Company
 .\"
-.Dd October 11, 2020
+.Dd November 15, 2023
 .Dt QSORT 3C
 .Os
 .Sh NAME
@@ -52,7 +52,7 @@ The contents of the table are sorted in ascending order according to the
 user-supplied comparison function.
 .Pp
 The
-.Fa Ibase
+.Fa base
 argument points to the element at the base of the table.
 The
 .Fa nel
@@ -62,6 +62,7 @@ The
 argument specifies the size of each element in bytes.
 The
 .Fa compar
+argument is the name of the comparison function, which is called with two
 arguments that point to the elements being compared.
 The comparison function need not compare every byte, so arbitrary data may be
 contained in the elements in addition to the values being compared.
@@ -94,7 +95,7 @@ The
 .Fn qsort
 and
 .Fn qsort_r
-function safely allows concurrent access by multiple threads
+functions safely allow concurrent access by multiple threads
 to disjoint data, such as overlapping subtrees or tables.
 .Sh EXAMPLES
 .Sy Example 1
-- 
2.43.0

